.gitignore

```
@@ -13,6 +13,7 @@ storybook-static
 
 # dependencies
 /node_modules
```

```suggestion
node_modules/
```
Would this do the same thing?

---

Yes! updated thanks 👍